dbscripts/xml/upgrade.xml

```diff
@@ -163,6 +163,8 @@
 		<migration class="PKP\migration\upgrade\v3_5_0\I7135_CreateNewRorRegistryCacheTables"/>
 		<migration class="PKP\migration\upgrade\v3_5_0\I10759_AddReviewAssignmentSettings"/>
 		<migration class="PKP\migration\upgrade\v3_5_0\I10819_OrcidOauthScopeMail"/>
+		<migration class="APP\migration\upgrade\v3_5_0\I4787_AddReviewSuggestionHelp"/>
```

Comment:
@touhidurabir I am not sure `I4787_AddReviewSuggestionHelp` and  `I4787_InstallReviewerSuggestion` are in the correct folder. I find them under `v3_6_0`

Reply:
Weird , I can see those migrations in `v3_5_0` folder .  Also tests seems to be passing as if in wrong folder , tests would have failed at upgrade stage for not being able to find the migration file .

Reply:
Maybe rebasing all branches onto main will solve that, along with the submodule conflicts? 

Just to clarify, I am always getting the PRs found here: https://github.com/pkp/pkp-lib/issues/4787#issue-447125184

Correct?

Reply:
@touhidurabir it was my bad, I correctly see the migrations in 3_5_0 now. Check this, though, if you can, regarding the conflicted submodules and json file. Thanks!!